tests/unit/utils.py

```diff
@@ -229,7 +229,7 @@ def target_path(self):
     @property
     def test_method_name(self):
         """ Test method name uses the original name. """
-        name = self.test_sub_path.split('.')[0]
+        name = os.path.splitext(self.test_sub_path)[0]
```

Comment:
i believe we need to also do name.replace('.', '_') since attributes cannot have a '.' in there name.

Reply:
hmm actually because we are since we are using getattr/setattr it is allowed but still I feel that test names should not have a '.' in them.

Reply:
Ack, will do.